hash.py

```diff
@@ -0,0 +1,45 @@
+class HashItem(object):
+    def __init__(self, key, value):
+        self.key = key
+        self.value = value
+
+
+class HashTable(object):
+    def __init__(self, tablesize=1024):
+        self.table = []
+        for i in range(tablesize):
+            self.table.append(list())
+
+    def hash(self, key):
+        hash_val = 0
+        for char in key:
+            hash_val += ord(char)
+        return hash_val % len(self.table)
```

Comment:
What if the thing provided as `key` is not a string?  What happens?  Do you have a test that demonstrates that?